src/api/app/helpers/webui/buildstatus_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Webui::BuildstatusHelper
+  # rubocop:disable Style/AlignHash
```

Comment:
Did you try to align the lines you moved to a newline with the indentation level of the other hash values? eg. line 6, 9, 16